media-video/pipewire/pipewire-0.3.23.ebuild

```diff
@@ -0,0 +1,229 @@
+		-Dsupport=true # Miscellaneous/common plugins, such as null sink
+		-Devl=false # Matches upstream
+		$(meson_use test) # Fake sink and source SPA for running tests
+		$(meson_use v4l v4l2)
+		-Dlibcamera=false # libcamera is not in Portage
+		-Dvideoconvert=true # Matches upstream
+		$(meson_use debug videotestsrc) # Generate a video test pattern
+		-Dvolume=false # Matches upstream
+		$(meson_use vulkan-spa vulkan) # Uses pre-compiled Vulkan compute shader to provide a CGI video source (looks like a dev thing)
+		$(meson_use extra pw-cat)
+		$(meson_use udev)
+#		-Dudevrulesdir="" # Directory for udev rules (defaults to /lib/udev/rules.d)
+#		-Dsystemd-user-unit-dir="" # Directory for user systemd units (defaults to pkg-config systemd --variable=systemduserunitpath ; prefix awareness expected but not verified)
+		$(meson_feature debug sdl2) # Enables SDL2 dependent code (currently only examples when -Dinstalled_tests=true)
+		$(meson_feature extra sndfile) # Enables libsndfile dependent code (currently only pw-cat)
+		# Gentoo added options
+		$(meson_feature aac)
+		$(meson_feature aptx)
+		$(meson_feature ldac)
+	)
+	meson_src_configure
+}
+
+src_compile() {
+	meson_src_compile
+
+	einfo "Generating ${limitsdfile}"
+	cat > ${limitsdfile} <<- EOF || die
+		# Start of ${limitsdfile} from ${P}
+
+		1000:60000	-	memlock 256
+
+		# End of ${limitsdfile} from ${P}
+	EOF
+}
+
+src_install() {
+	meson_src_install
+
+	insinto /etc/security/limits.d/
+	doins ${limitsdfile}
+
+	if use pipewire-alsa; then
+		keepdir /etc/alsa/conf.d/
+
+		dosym8 -r /usr/share/alsa/alsa.conf.d/50-pipewire.conf /etc/alsa/conf.d/50-pipewire.conf
+		dosym8 -r /usr/share/alsa/alsa.conf.d/99-pipewire-default.conf /etc/alsa/conf.d/99-pipewire-default.conf
+	fi
+}
+
+pkg_postinst() {
+	elog "It's recommended to install and enable sys-auth/rtkit daemon to allow"
+	elog "PipeWire to obtain restricted realtime capabilities at runtime."
+	elog
+	if use pipewire-jack; then
+		elog "JACK support is incomplete and not all programs will work."
+		elog "PipeWire's alternative libraries have been installed to a"
+		elog "non-default location; to use them, prepend pw-jack <application>"
+		elog "before every JACK application invocation."
+		elog
+	fi
+	if use systemd; then
+		elog "Per Gentoo policy installed systemd units must be manually enabled:"
+		elog "systemctl --user disable pulseaudio.service pulseaudio.socket"
+		elog "systemctl --user enable pipewire.socket pipewire-pulse.socket"
+		elog "systemctl --user enable pipewire-media-session.service"
+		elog
+		elog "Rebooting is strongly recommended to avoid surprises from"
+		elog "remnant PulseAudio daemon auto-spawning and surviving logouts."
+		elog
+	else
+		elog "This ebuild tries to do right by OpenRC users by auto-enabling"
+		elog "PulseAudio replacement."
+		elog "For now, users are strongly advised to edit:"
+		elog "${EROOT}/etc/pulse/client.conf"
+		elog "by uncommenting the following line and seting its value to no:"
```

Comment:
suggestion: and to uncomment the following line 
typo: setting

p.s.: why uncommenting and setting value to no?

Reply:
Because PulseAudio like systemd and OpenSSH lists its default values commented out and lets people put in the values they want themselves, so to disable auto-spawn'ing of the pulseaudio daemon the following line:
_; autospawn = yes_
must be turned into:
_autospawn = no_

I felt that using a present participle made the text flow better but if I get the feeling or a GLEP that elog messages should be as simple and easy to understand as possible, then I'll look into turning them into something more like simple English.

As for the typo, sure, I'll fix that. Well spotted.

Reply:
I think elog messages should be kept as simple as possible, this at least if you want to include non nativ speakers.